.ado/templates/npm-publish-steps.yml

```diff
@@ -41,7 +41,9 @@ steps:
       condition: and(succeeded(), eq(variables['publish_react_native_macos'], '1'))
 
     - script: |
-        yarn nx release publish --excludeTaskDependencies
+        if [[ -f .rnm-publish ]]; then
+          yarn nx release publish --excludeTaskDependencies
+        fi
```

Comment:
Should we also remove the file? Not sure it matters at this point.

Reply:
Unless ADO keeps random files around between runs, I think this should be safe to ignore. There's nothing in it that could be abused in later steps.